### PR TITLE
chore: Replicache 13, remove replicache-express dependency

### PR DESCRIPTION
**Pull request by cesara**

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -1,40 +1,82 @@
-import 'dotenv/config';
-import path from 'path';
-import {ReplicacheExpressServer} from 'replicache-express';
-import {mutators} from 'shared';
-import {fileURLToPath} from 'url';
+import type {MutatorDefs} from 'replicache';
```
Comment (aboodman) [inline server/src/server.ts]: I think this file can likely go away and just be merged into main. There's no need for the abstraction anymore (this used to be part of `replicache-express`.

File: server/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import type Express from 'express';
+import {createSpace as createSpaceImpl, getCookie} from './data.js';
+import {handleRequest as handleRequestImpl} from '../endpoints/handle-request.js';
+import {transact} from './pg.js';
+import type {MutatorDefs} from 'replicache';
+
+export async function spaceExists(spaceID: string) {
```
Comment (aboodman) [inline server/src/index.ts]: How does this function relate to the one in `handle-space.ts`?

File: server/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import type Express from 'express';
```
Comment (aboodman) [inline server/src/index.ts]: I think probably this file can go away too ... most of it is just delegating to endpoints at this point which can be done by `main.ts`.

File: server/src/index.ts (hunk 2 above)
Comment (cesara) [reply server/src/index.ts]: redundant. streamlined this as well

**Pull request merged**